renderer/configuration/robotouille_config.json

```diff
@@ -1,6 +1,5 @@
 {
```

Comment:
can you clean up this JSON to retain the original formatting that your linter modified so only relevant changes show.

Reply:
fixed